lsp/src/main/java/lsp/LSPServer.java

```diff
@@ -75,6 +75,15 @@ public LSPServer(Dialect dialect, InputStream inStream, OutputStream outStream)
 		System.setProperty("vdmj.debug.link_class", DAPDebugLink.class.getName());
 		Settings.annotations = true;
 		Settings.dialect = dialect;
+
+		String svalue = System.getProperty("vdmj.strict");
+		if ( svalue != null )
+			Settings.strict = Boolean.parseBoolean(svalue);
```

Comment:
There is a Boolean.getBoolean("property.name") method which may help with this? It is probably reasonable to call this property "vdmj.strict" since it isn't an LSP thing.